### ISSUE TITLE
Step 3 - Create Sample Task - Fails

### ISSUE THREAD

**Issue by weaselchops** [openshift-labs/learn-katacoda#1137]
Hi,

Going through the https://learn.openshift.com/middleware/pipelines/ lab. today
Step 3 Create Sample Task, fails
OC 4.7.
I have restarted this lab a few times, using different provided clusters.

```
$ echo -e "Welcome to your interactive environment. OpenShift is configured and ready to use."
Welcome to your interactive environment. OpenShift is configured and ready to use.
$ 
$ 
$ until oc api-resources --api-group=tekton.dev | grep tekton.dev &> /dev/null
> do 
>  echo "Operator installation in progress..."
>  sleep 5
> done
$ 
$ echo "Operator ready"
Operator ready
$ oc new-project pipelines-tutorial
Now using project "pipelines-tutorial" on server "https://openshift:6443".

You can add applications to this project with the 'new-app' command. For example, try:

    oc new-app rails-postgresql-example

to build a new example application in Ruby. Or use kubectl to deploy a simple Kubernetes application:

    kubectl create deployment hello-node --image=k8s.gcr.io/serve_hostname

$ oc apply -f tasks/hello.yaml
task.tekton.dev/hello created
$ tkn task start --showlog hello
TaskRun started: hello-run-7vswh
Waiting for logs to be available...
task hello create has not started yet or pod for task not yet available
$
```
On the OC console
<Clster address>tasks/ns/pipelines-tutorial/task-runs
TypeError
Description:
Cannot read property 'tekton.dev/pipeline' of undefined

**Component Trace:**
```
in m
    in s
    in H
    in div
    in div
    in t
    in t
    in o
    in div
    in $
    in div
    in Data
    in StatusBox
    in div
    in a
    in withI18nextTranslation(a)
    in O
    in Unknown
    in div
    in div
    in div
    in ListPageWrapper_
    in Unknown
    in Unknown
    in O
    in Unknown
    in O
    in div
    in Unknown
    in O
    in O
    in MultiListPage
    in Unknown
    in o
    in ListPage
    in Unknown
    in t
    in t
    in div
    in Unknown
    in Unknown
    in Unknown
    in div
    in div
    in Unknown
    in Unknown
    in l
    in t
    in C
    in t
    in div
    in div
    in section
    in PageSection
    in Unknown
    in div
    in div
    in DrawerMain
    in DrawerContent
    in div
    in Drawer
    in NotificationDrawer
    in Unknown
    in O
    in main
    in div
    in Page
    in div
    in DrawerContentBody
    in div
    in div
    in DrawerMain
    in DrawerContent
    in div
    in Drawer
    in Unknown
    in L
    in L
    in Unknown
    in s
    in Unknown
    in withExtensions()
    in withI18nextTranslation(withExtensions())
    in t
    in t
    in t
    in t
    in Suspense
```
**Stack Trace**
```
TypeError: Cannot read property 'tekton.dev/pipeline' of undefined
    at t.a (https://console-openshift-console-2886795281-443-hazel05.environments.katacoda.com/static/admin-tasks%60-chunk-94c30cc17e6eb949c5d6.min.js:11:63629)
    at rowRenderer (https://console-openshift-console-2886795281-443-hazel05.environments.katacoda.com/static/main-chunk-bee1364c44ecb41ce95e.min.js:1:1061777)
    at _cellRenderer (https://console-openshift-console-2886795281-443-hazel05.environments.katacoda.com/static/vendors~main-chunk-e31d56a926ed1b83fb48.min.js:268:440272)
    at r (https://console-openshift-console-2886795281-443-hazel05.environments.katacoda.com/static/vendors~main-chunk-e31d56a926ed1b83fb48.min.js:216:81057)
    at m._calculateChildrenToRender (https://console-openshift-console-2886795281-443-hazel05.environments.katacoda.com/static/vendors~main-chunk-e31d56a926ed1b83fb48.min.js:216:71268)
    at m.render (https://console-openshift-console-2886795281-443-hazel05.environments.katacoda.com/static/vendors~main-chunk-e31d56a926ed1b83fb48.min.js:216:68498)
    at va (https://console-openshift-console-2886795281-443-hazel05.environments.katacoda.com/static/vendors~main-chunk-e31d56a926ed1b83fb48.min.js:244:65523)
    at ma (https://console-openshift-console-2886795281-443-hazel05.environments.katacoda.com/static/vendors~main-chunk-e31d56a926ed1b83fb48.min.js:244:65318)
    at Ys (https://console-openshift-console-2886795281-443-hazel05.environments.katacoda.com/static/vendors~main-chunk-e31d56a926ed1b83fb48.min.js:244:99601)
    at Bs (https://console-openshift-console-2886795281-443-hazel05.environments.katacoda.com/static/vendors~main-chunk-e31d56a926ed1b83fb48.min.js:244:84008)
```

**Comment by azaceta** [comment]
Same issue here, I started suspecting of a configMap for the SSL cert not being available, but I'm at a lost, this is getting from the openshift-pipelines-operator logs:
`[{"type":"DependenciesInstalled","status":"Unknown","lastTransitionTime":"2021-06-07T10:58:32Z"},{"type":"DeploymentsAvailable","status":"Unknown","lastTransitionTime":"2021-06-07T10:58:32Z"},{"type":"InstallSucceeded","status":"False","lastTransitionTime":"2021-06-07T10:59:32Z","reason":"Error","message":"Install failed with message: tektonpipeline pipeline is not in desired state, got: <nil>: timed out waiting for the condition: For more info Please check TektonPipeline CR status"},{"type":"Ready","status":"False","lastTransitionTime":"2021-06-07T10:59:32Z","reason":"Error","message":"Install failed with message: tektonpipeline pipeline is not in desired state, got: <nil>: timed out waiting for the condition: For more info Please check TektonPipeline CR status"}]},"logging.googleapis.com/labels":{},"logging.googleapis.com/sourceLocation":{"file":"/opt/app-root/src/go/src/github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektonconfig/tektonconfig.go","line":"99","function":"github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektonconfig.(*Reconciler).ReconcileKind"}}`
Not sure about next steps.

**Comment by blues-man** [comment]
Probably a temporary issue? I can't reproduce it

**Comment by azaceta** [comment]
This was solved to me with the workaround in #1164

**Comment by blues-man** [comment]
Ok cool, thanks @azaceta ! :) Closing this issue then
